editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1514,6 +1510,10 @@ export const MetadataUtils = {
     if (isImported) {
       return false
     }
+    const elementName = MetadataUtils.getJSXElementName(maybeEitherToMaybe(element?.element))
```

Comment:
Super minor, but this line could even be moved after the below conditional

Reply:
Fixed in 0330ba2b4.